src/components/TrialBanner.js

```diff
@@ -0,0 +1,196 @@
+        div({ style: { flexGrow: 1, marginBottom: '0.5rem' } }),
+        h(LabeledCheckbox, {
+          checked: cloudTermsAgreed === true,
+          onChange: v => this.setState({ cloudTermsAgreed: v })
+        }, [
+          span({ style: { marginLeft: '0.5rem' } }, [
+            'I agree to the Google Cloud Terms of Service.', div({ style: { marginLeft: '1.5rem' } }, [
+              'Google Cloud Terms of Service:',
+              a({
+                style: { textDecoration: 'underline', marginLeft: '0.25rem' },
+                target: '_blank',
+                href: 'https://cloud.google.com/terms/'
+              }, ['https://cloud.google.com/terms/', icon('pop-out', { style: { marginLeft: '0.25rem' } })])
+            ])
+          ])
+        ])
+      ])
+    ])
+  }
+
+  render() {
+    const { authState: { isSignedIn, profile }, ajax: { User } } = _.omit('isVisible', this.props)
+    const { accessingCredits, loading, finalizeTrial } = this.state
+    const { trialState } = profile
+    if (!trialState || !isSignedIn || trialState === 'Finalized') return null
+    const { [trialState]: { title, message, enabledLink, button, isWarning } } = messages
+
+    return div([
+      div({
+        style: {
+          display: 'flex', alignItems: 'center', padding: '1.5rem', height: 110,
+          backgroundColor: isWarning ? colors.orange[0] : '#359448',
+          justifyContent: 'center', color: 'white', width: '100%', fontSize: '1rem'
+
+        }
+      },
+      [
+        div({
+          style: {
+            fontSize: '1.5rem', fontWeight: 500, textAlign: 'right', borderRight: '1px solid', paddingRight: '1rem', marginRight: '1rem',
+            maxWidth: 200, flexShrink: 0
+          }
+        }, title),
+        span({ style: { maxWidth: 600, lineHeight: '1.5rem' } },
+          [
+            message,
+            enabledLink && a({
+              style: { textDecoration: 'underline', marginLeft: '0.5rem' },
+              target: '_blank',
+              href: enabledLink.url
+            }, [enabledLink.label, icon('pop-out', { style: { marginLeft: '0.25rem' } })])
+          ]),
+        h(Clickable, {
+          style: {
+            fontWeight: 500, fontSize: '1.125rem', border: '2px solid', borderRadius: '0.25rem', padding: '0.5rem 1rem',
+            marginLeft: '0.5rem', flexShrink: 0
+          },
+          onClick: () => {
+            button.isExternal ? window.open(button.url, '_blank') : this.setState({ accessingCredits: true })
+          }
+        }, [
+          loading ? spinner({ style: { fontSize: '1rem', color: 'white' } }) : button.label,
+          button.isExternal ? icon('pop-out', { style: { marginLeft: '0.25rem' } }) : null
+        ]),
+        div({ style: { marginLeft: '3rem', display: 'flex', flexDirection: 'column', alignItems: 'flex-end' } }, [
+          (trialState === 'Terminated') && h(Clickable, {
+            style: { borderBottom: 'none' },
+            tooltip: 'Hide forever?',
```

Comment:
I think the tooltip is not necessary since there is a confirmation dialog once someone clicks on the X.